test/rfc7540_SUITE.erl

```diff
@@ -2975,8 +2976,10 @@ client_settings_disable_push(Config) ->
 %   GOAWAY frame with an updated last stream identifier.  This ensures
 %   that a connection can be cleanly shut down without losing requests.
 %
-%% @todo And of course even if we shutdown we need to be careful about
-%% the connection state.
+%% And of course even if we shutdown we need to be careful about the
+%% connection state. In state 'closing', we don't accept new streams
+%% because we have already sent a GOAWAY frame with the last stream ID.
```

Comment:
Just remove the todo. Remove the todo comments in general for things that have been implemented and tested in this test suite.